juriscraper/pacer/attachment_page.py

```diff
@@ -15,11 +15,14 @@ class AttachmentPage(BaseReport):
 
     PATH = "doc1/"
 
-    def __init__(self, court_id, pacer_session=None):
+    def __init__(
+        self, court_id, pacer_session=None, notification_att_page=False
+    ):
         super().__init__(court_id, pacer_session)
         # Note that parsing bankruptcy attachment pages does not reveal the
         # document number, only the attachment numbers.
         self.is_bankruptcy = self.court_id.endswith("b")
+        self.is_notification_att_page = notification_att_page
```

Comment:
I'm a bit confused here. Are the NEF att pages different from the regular ones? They look pretty much the same to me?

Reply:
Yeah, I added this for "nyed" because the extra column was only on the attachment page from notifications. Since we are going to count columns, this wouldn't be needed anymore.

Reply:
Well, not just that, but why do we need an `nef_attachment_pages` test directory? Aren't those the same as the attachment_pages tests?

Reply:
Removed.